doc/release-notes.rst

```diff
@@ -163,7 +163,8 @@ Major Changes from Kraken
   * Specifying user authorization capabilities for RBD clients has been
     simplified. The general syntax for using RBD capability profiles is
     "mon 'profile rbd' osd 'profile rbd[-read-only][ pool={pool-name}[, ...]]'".
-    For more details see "User Management" in the documentation.
+    For more details see :doc:`rados/operations/user-management/`
```

Comment:
rados/operation/user-management does not looks to be directory.
this is file user-management.rst

Reply:
The ":doc:" tag used throughout this file just links to the directories that are created when the ```.rst``` files are processed.